src/pages/iou/request/step/IOURequestStepDistanceRate.tsx

```diff
@@ -78,10 +76,12 @@ function IOURequestStepDistanceRate({
     const initiallyFocusedOption = sections.find((item) => item.isSelected)?.keyForList;
 
     function selectDistanceRate(customUnitRateID: string) {
-        if (policy?.customUnits && customUnitID && shouldShowTax) {
```

Comment:
For distance `policy?.customUnits && customUnitID` are always set, so I found it redundant and hence removed and cleaned up the code